ingestion_pipeline_steps.py----ingestion_pipeline_steps---2024-09-11 19:51:36,890---INFO---data_ingestion_pipeline---Hate_classification.LOGGERS.logging---Initializing  data ingestion pipeline steps
data_ingestion.py----data_ingestion---2024-09-11 19:51:36,890---INFO---ingest_data---Hate_classification.LOGGERS.logging---initialized Reading CSV from { self.path} 
data_ingestion.py----data_ingestion---2024-09-11 19:51:36,890---INFO---ingest_data---Hate_classification.LOGGERS.logging---initialized Reading CSV from { self.path} 
ingestion_pipeline_steps.py----ingestion_pipeline_steps---2024-09-11 19:51:36,936---INFO---data_ingestion_pipeline---Hate_classification.LOGGERS.logging---successfully finished data ingestion pipeline steps
ingestion_pipeline_steps.py----ingestion_pipeline_steps---2024-09-11 19:51:36,936---INFO---data_ingestion_pipeline---Hate_classification.LOGGERS.logging---successfully finished data ingestion pipeline steps
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,961---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---Initializing data processing pipeline 
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,961---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---Initializing data processing pipeline 
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,961---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---Initializing data processing pipeline 
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---initializing the dropping of the columns
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---initializing the dropping of the columns
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---initializing the dropping of the columns
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---initializing the dropping of the columns
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---Columns available in DataFrame: ['Unnamed: 0', 'count', 'hate_speech', 'offensive_language', 'neither', 'class', 'tweet']
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---Columns available in DataFrame: ['Unnamed: 0', 'count', 'hate_speech', 'offensive_language', 'neither', 'class', 'tweet']
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---Columns available in DataFrame: ['Unnamed: 0', 'count', 'hate_speech', 'offensive_language', 'neither', 'class', 'tweet']
data_processing.py----data_processing---2024-09-11 19:51:36,962---INFO---preprocess_data---Hate_classification.LOGGERS.logging---Columns available in DataFrame: ['Unnamed: 0', 'count', 'hate_speech', 'offensive_language', 'neither', 'class', 'tweet']
data_processing.py----data_processing---2024-09-11 19:51:36,965---INFO---preprocess_data---Hate_classification.LOGGERS.logging---successfully dropped the columns
data_processing.py----data_processing---2024-09-11 19:51:36,965---INFO---preprocess_data---Hate_classification.LOGGERS.logging---successfully dropped the columns
data_processing.py----data_processing---2024-09-11 19:51:36,965---INFO---preprocess_data---Hate_classification.LOGGERS.logging---successfully dropped the columns
data_processing.py----data_processing---2024-09-11 19:51:36,965---INFO---preprocess_data---Hate_classification.LOGGERS.logging---successfully dropped the columns
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,971---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---dropped columns successfully
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,971---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---dropped columns successfully
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,971---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---dropped columns successfully
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,971---INFO---data_processing_pipeline---Hate_classification.LOGGERS.logging---dropped columns successfully
data_processing.py----data_processing---2024-09-11 19:51:36,976---INFO---lower_case---Hate_classification.LOGGERS.logging---error applying the lowercase transformation - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 67, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,976---INFO---lower_case---Hate_classification.LOGGERS.logging---error applying the lowercase transformation - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 67, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,976---INFO---lower_case---Hate_classification.LOGGERS.logging---error applying the lowercase transformation - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 67, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,976---INFO---lower_case---Hate_classification.LOGGERS.logging---error applying the lowercase transformation - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 67, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,976---INFO---lower_case---Hate_classification.LOGGERS.logging---error applying the lowercase transformation - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 67, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,980---INFO---drop_punctuations---Hate_classification.LOGGERS.logging---error dropping punctuations and urls - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 78, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,980---INFO---drop_punctuations---Hate_classification.LOGGERS.logging---error dropping punctuations and urls - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 78, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,980---INFO---drop_punctuations---Hate_classification.LOGGERS.logging---error dropping punctuations and urls - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 78, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,980---INFO---drop_punctuations---Hate_classification.LOGGERS.logging---error dropping punctuations and urls - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 78, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,980---INFO---drop_punctuations---Hate_classification.LOGGERS.logging---error dropping punctuations and urls - Error: Item wrong length 2 instead of 24783. has occurred in module: ValueError, line: 78, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,981---ERROR---preprocess_data---Hate_classification.LOGGERS.logging---there was an error while perfoming the normalization steps - Error: 'Logger' object is not callable has occurred in module: TypeError, line: 54, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,981---ERROR---preprocess_data---Hate_classification.LOGGERS.logging---there was an error while perfoming the normalization steps - Error: 'Logger' object is not callable has occurred in module: TypeError, line: 54, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,981---ERROR---preprocess_data---Hate_classification.LOGGERS.logging---there was an error while perfoming the normalization steps - Error: 'Logger' object is not callable has occurred in module: TypeError, line: 54, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,981---ERROR---preprocess_data---Hate_classification.LOGGERS.logging---there was an error while perfoming the normalization steps - Error: 'Logger' object is not callable has occurred in module: TypeError, line: 54, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing.py----data_processing---2024-09-11 19:51:36,981---ERROR---preprocess_data---Hate_classification.LOGGERS.logging---there was an error while perfoming the normalization steps - Error: 'Logger' object is not callable has occurred in module: TypeError, line: 54, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\COMPONENTS\data_processing.py
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,981---ERROR---data_processing_pipeline---Hate_classification.LOGGERS.logging---error occured while perfoming normalization data - Error: 'NoneType' object has no attribute 'head' has occurred in module: AttributeError, line: 27, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\PIPELINE\data_processing_pipeline.py
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,981---ERROR---data_processing_pipeline---Hate_classification.LOGGERS.logging---error occured while perfoming normalization data - Error: 'NoneType' object has no attribute 'head' has occurred in module: AttributeError, line: 27, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\PIPELINE\data_processing_pipeline.py
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,981---ERROR---data_processing_pipeline---Hate_classification.LOGGERS.logging---error occured while perfoming normalization data - Error: 'NoneType' object has no attribute 'head' has occurred in module: AttributeError, line: 27, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\PIPELINE\data_processing_pipeline.py
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,981---ERROR---data_processing_pipeline---Hate_classification.LOGGERS.logging---error occured while perfoming normalization data - Error: 'NoneType' object has no attribute 'head' has occurred in module: AttributeError, line: 27, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\PIPELINE\data_processing_pipeline.py
data_processing_pipeline.py----data_processing_pipeline---2024-09-11 19:51:36,981---ERROR---data_processing_pipeline---Hate_classification.LOGGERS.logging---error occured while perfoming normalization data - Error: 'NoneType' object has no attribute 'head' has occurred in module: AttributeError, line: 27, file: C:\Users\User\Desktop\nlp_text_class\Hate_classification\PIPELINE\data_processing_pipeline.py
